tests/.eslintrc.yml

```diff
@@ -1,4 +1,4 @@
 env:
-  mocha: true
+    mocha: true
 rules:
-  node/no-unpublished-require: off # we’re using devDeps here.
+    node/no-unpublished-require: off # we’re using devDeps here.
```

Comment:
base on `.editorconfig` yml should have 2 spaces but its overridden by `.prettierrc`